pkg/server/image_pull.go

```diff
@@ -346,6 +353,17 @@ func (c *criService) registryHosts(auth *runtime.AuthConfig) docker.RegistryHost
 	}
 }
 
+// defaultScheme returns the default scheme for a registry host.
+func defaultScheme(host string) (string, error) {
```

Comment:
probably don't need to return error since it's always nil..

Reply:
Done. Left from old change.